packages/expo-speech/src/Speech/Speech.ts

```diff
@@ -61,6 +61,10 @@ export function speak(text: string, options: SpeechOptions = {}) {
   ExponentSpeech.speak(String(id), text, options);
 }
 
+export async function getAvailableVoicesAsync() {
+  return ExponentSpeech.getVoices();
```

Comment:
We might need to check whether `ExponentSpeech.getVoices` exists and throw `UnavailabilityError` if not.